src/server/polaris-server.ts

```diff
@@ -115,6 +137,21 @@ export class PolarisServer {
         };
     }
 
+    private getSchemaWithMiddlewares(): GraphQLSchema {
```

Comment:
i think that you should do here like you did with the plugins, create a private method that takes the relevant middlewares(including the custom and the snapshot.

Reply:
resolved on whatsapp